src/map/status.cpp

```diff
@@ -10858,909 +9189,347 @@ int status_change_start(struct block_list* src, struct block_list* bl,enum sc_ty
 	undead_flag = battle_check_undead(status->race,status->def_ele);
 	// Check for immunities / sc fails
 	switch (type) {
```

Comment:
Becomes uncomparable below here because tabbing was changed by 1.
Hopefully everything is still good!